src/test/java/uk/gov/hmcts/cmc/claimstore/events/utils/sampledata/SampleClaimIssuedEvent.java

```diff
@@ -12,6 +12,8 @@ private SampleClaimIssuedEvent() {
     }
 
     public static final String CLAIMANT_EMAIL = SampleClaim.SUBMITTER_EMAIL;
+    public static final String SUBMITTER_SURNAME = "Patrick";
```

Comment:
Can we please use a proper surname, not something that looks like a forename?

Reply:
actioned